src/util.h

```diff
@@ -38,6 +38,10 @@
  * This should be the size of the buffer given to ld2string */
 #define MAX_LONG_DOUBLE_CHARS 5*1024
 
+/* The length of an integer is no more than 20.
+ * This could be the limit for string2ll to save execution time. */
+#define MAX_LONG_LONG_LEN 20
```

Comment:
at first, i wasn't certain how this define is useful for others, and wanted to suggest putting it in the C file where it's used rather than the header.

The i realized we have `MAX_LONG_DOUBLE_CHARS` right above it, which is used by callers of `ld2string` to know the size of buffer to allocate, so that could be useful.

looking for callers to `ll2string` i see many use a const of 32 or 128, but some actually use `LONG_STR_SIZE` (which also has space for the null terminator).

so maybe we need t move that define rather than define an additional one.

Reply:
i suppose i presented too many options, and i need to state which one i'd prefer.
i guess that would be be either to move LONG_STR_SIZE to util.h (keeping it's name and comment).